docs/core.rst

```diff
@@ -45,13 +46,18 @@ Barycenters (Entropic and LR)
     :toctree: _autosummary
 
     discrete_barycenter.discrete_barycenter
+    continuous_barycenter.WassersteinBarycenter
+    continuous_barycenter.BarycenterState
+    gw_barycenter.GromovWassersteinBarycenter
```

Comment:
maybe worth adding a few refs in docs on this, probably our GW barycenter paper + the FGW work by Vayer Titouan, Chapel Laetitia, Flamary Rémi, Tavenard Romain and Courty Nicolas “Optimal Transport for structured data with application on graphs” International Conference on Machine Learning (ICML). 2019.

Reply:
I've introduced bibtex files to the documentation and added those references. In the future, would convert all other references to `.bib` files.